### ISSUE TITLE
Allow passing null for things like activeClass so router-link doesn't render attribute

### ISSUE THREAD

**Issue by jez9999** [vuejs/router#1546]
### What problem does this feature solve?
At the moment, you can pass in empty strings to prevent a value being rendered in `router-link` for when a link is active:

```
<router-link active-class="" exact-active-class="" aria-current-value="">Home</router-link>
```

However, when the link is active, this causes an empty 'class' and 'aria-current' attribute to be rendered.  It would be nice to remove the attributes altogether.

### What does the proposed API look like?
You should be able to pass `null` as a JS expression to remove the attributes altogether when the link is active:

```
<router-link :active-class="null" :exact-active-class="null" :aria-current-value="null">Home</router-link>
```

**Comment by posva** [comment]
what is the use case for this?

**Comment by jez9999** [comment]
Clean DOM?  I just don't like having empty attributes hanging around.

**Comment by VincentVrijburg** [comment]
I just found this issue and have another use case for it.

I have a link on a child route page that points to the parent route. This is just an arrow pointing back and is now (by default) marked as active. When I set the router-link attributes to empty (as stated above) an empty class and aria-current are still being rendered.

I guess it's fine to have those empty attributes but I would appreciate it if it was possible to set something on router links to not consider them for active marking.
